src/lib/aws/glue_manager.py

```diff
@@ -17,4 +59,19 @@ def get_all_job_names(self):
         except Exception as e:
             error_message = f"Error getting list of glue jobs : {e}"
             raise GlueManagerException(error_message)            
+        
+    def get_job_runs(self, job_name: str, since_time: datetime) -> list[JobRun]:
+        try:
+            outp = []
```

Comment:
This row is excess. outp will be set to an empty list below if the condition "x.StartedOn > since_time" is False.

Reply:
fixed